modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PageMemoryImplNoLoadTest.java

```diff
@@ -130,7 +130,8 @@
                     return true;
                 }
             },
-            new DataRegionMetricsImpl(new DataRegionConfiguration(), cctx.metric(), NO_OP_METRICS),
+            new DataRegionMetricsImpl(
```

Comment:
Should we add `PerformanceStatistics` instance to ctx on line 68?

Reply:
done